dataans/src/app_info/import.rs

```diff
@@ -0,0 +1,43 @@
+use leptos::*;
+
+use crate::backend::import::{import_app_data, select_file};
+
+#[component]
+pub fn Import() -> impl IntoView {
+    let toaster = leptoaster::expect_toaster();
+    let (is_importing, set_is_importing) = create_signal(false);
+
+    let import_data = move |_| {
+        if is_importing.get() {
+            return;
+        }
```

Comment:
Do you really need to check this? I mean, the import button is disabled during importing anyway (`disabled=is_importing.get()`)